November Multilingual Publishing Comm Call - English (#607)
ropensci/roweb3: content/commcalls/2023-10-31/index.md

---

> title: "R in Goverment"

Author: ```suggestion
title: "Multilingual Publishing"
```

> dateStart: 2023-10-21T12:00:00 # UTC!!

Author: ```suggestion
dateStart: 2023-11-23T12:00:00 # UTC!!
```

dateEnd: 2023-10-21T13:00:00 # UTC!!

> date: 2023-10-21T13:00:00 # UTC!! same as dateEnd

Author: ```suggestion
date: 2023-11-23T13:00:00 # UTC!! same as dateEnd
```

> publishDate: 2023-08-19

Author: ```suggestion
publishDate: 2023-08-03
```

description: 
author:
  - 
  - 
  - 
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: nov2023-multilingual
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:

>   - url: 

Author: ```suggestion
  - url: https://devdevguide.netlify.app/es/preface.es.html
```

>     name:   

Author: ```suggestion
    name:   rOpenSci DEvelopment Guide in Spanish
```

>     details:  

Author: ```suggestion
    details:  First version of the rOpenSci Packages: Development, Maintenance, and Peer Review in Spanish
```

  - url: 
    name: 

>   - url: /champions/

Author: ```suggestion
  - url: /multilingual-publishing/
```

>     name:   Champions Program

Author: ```suggestion
    name:   Multilingual Publishing
```

>     details: web page with details about the champion program.

Author: ```suggestion
    details: web page with details about the Multilingual Publishing project.
```

speakers:  

>   - name: 

Reviewer: ```suggestion
  - name: Pao Corrales
```

>     bio: 

Reviewer: ```suggestion
    bio: Pao is a PhD candidate in atmospheric sciences working on severe weather forecasts in Argentina. She is also a professor at Universidad Nacional Guillermo Brown teaching R and related tools. She also develops open-licensed teaching materials and contributes to several communities of practise related to R.
```

>   - url: 

Author: ```suggestion
  - url: https://docs.ropensci.org/babeldown/
```

>     name: 

Author: ```suggestion
    name: babeldown R package
    details: Translate Markdown-based content via DeepL API
```

>     img: /img/team/beatriz-milz-mentor.jpeg

Reviewer: ```suggestion
    img: /img/team/paola-corrales.jpeg
```

>   - name: 
>     bio:         
>     img: /img/team/victor-ordu-mentee.jpg  

Reviewer 2: ```suggestion
  - name: Elio Campitelli
    bio: "Elio is a a PhD student in atmospheric sciences at the Centre for Ocean and Atmospheric Research. They also maintain several open-source R packages (e.g., ggnewscale; metR) and contribute to other packages, such as data.table and ggplot2. Follow them on [mastodon](https://mastodon.social/@eliocamp) and check their [website](https://eliocamp.github.io/)."
    img: /img/team/elio-campitelli.jpg
```

  - name: Maëlle Salmon
    bio: "Maëlle Salmon is a R(esearch) Software Engineer, part-time with rOpenSci where she, among other things, created and maintains the babeldown and babelquarto R packages, and maintains the guide [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/). She also created the [R-hub blog](https://blog.r-hub.io) and co-wrote the book [HTTP testing in R](https://books.ropensci.org/http-testing) with [Scott Chamberlain](/author/scott-chamberlain). She lives in Nancy, France. She's an enthusiastic polyglot (of Latin and Germanic languages). Maëlle on [GitHub](https://github.com/maelle), [Mastodon](https://mastodon.social/@maelle), [Website](https://masalmon.eu/), [rOpenSci](/author/maëlle-salmon/)."
    img: /img/team/maelle_salmon.jpg
tags:
  - community call
  - events
  - multilingual
deets: |

>     Meeting ID:  875-494-77466

Author: ```suggestion
    Meeting ID:  838-6678-3222
```

    Passcode: 516675

> zoomurl: https://numfocus-org.zoom.us/j/87549477466?pwd=b0cydVpIYW5yMm9MKzZONG16RTVsUT09

Author: ```suggestion
zoomurl: https://numfocus-org.zoom.us/j/83866783222?pwd=Y1k3OEJnakc3WVRISlZxQ3NTUU1yZz09
```

---
title: "Multilingual Publishing"
dateStart: 2023-11-23T12:00:00 # UTC!!
dateEnd: 2023-11-23T13:00:00 # UTC!!
date: 2023-11-23T13:00:00 # UTC!! same as dateEnd
publishDate: 2023-08-03
description: 
author:
  - 
  - 
  - 
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: nov2023-multilingual
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:
  - url: https://devdevguide.netlify.app/es/preface.es.html
    name:   rOpenSci DEvelopment Guide in Spanish
    details:  First version of the rOpenSci Packages: Development, Maintenance, and Peer Review in Spanish
  - url: https://docs.ropensci.org/babeldown/
    name: babeldown R package
    details: Translate Markdown-based content via DeepL API

>   - url: /multilingual-publishing/

Author: ```suggestion
  - url: https://docs.ropensci.org/babelquarto/
    name: babelquarto R package
    details: Set up and render a multilingual Quarto book
  - url: /multilingual-publishing/
```

    img: /img/team/elio-campitelli.jpg
  - name: Maëlle Salmon
    bio: "Maëlle Salmon is a R(esearch) Software Engineer, part-time with rOpenSci where she, among other things, created and maintains the babeldown and babelquarto R packages, and maintains the guide [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/). She also created the [R-hub blog](https://blog.r-hub.io) and co-wrote the book [HTTP testing in R](https://books.ropensci.org/http-testing) with [Scott Chamberlain](/author/scott-chamberlain). She lives in Nancy, France. She's an enthusiastic polyglot (of Latin and Germanic languages). Maëlle on [GitHub](https://github.com/maelle), [Mastodon](https://mastodon.social/@maelle), [Website](https://masalmon.eu/), [rOpenSci](/author/maëlle-salmon/)."
    img: /img/team/maelle_salmon.jpg
tags:
  - community call
  - events
  - multilingual
deets: |
    Meeting ID:  838-6678-3222
    Passcode: 584914
zoomurl: https://numfocus-org.zoom.us/j/83866783222?pwd=Y1k3OEJnakc3WVRISlZxQ3NTUU1yZz09
---

> 

Author: ```suggestion
Open Source and Open Science, as global movements, faces language-based exclusion as most resources are in English. This affects scientists and research software engineers (RSE) working in R, particularly those who don't have English as first language. 

rOpenSci multilingual efforts aim to lower access barriers, democratize quality resources, and increase the possibilities of contributing to open software and science. We successfully piloted our Spanish-language peer review and the localization to Spanish of our comprehensive guide to software development, with Portuguese translation underway.
```

---
title: "Multilingual Publishing"
dateStart: 2023-11-23T12:00:00 # UTC!!
dateEnd: 2023-11-23T13:00:00 # UTC!!
date: 2023-11-23T13:00:00 # UTC!! same as dateEnd
publishDate: 2023-08-03
description: 
author:
  - 
  - 
  - 
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: nov2023-multilingual
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:
  - url: https://devdevguide.netlify.app/es/preface.es.html

>     name:   rOpenSci DEvelopment Guide in Spanish

Author: ```suggestion
    name: rOpenSci DEvelopment Guide in Spanish
```

    img: /img/team/maelle_salmon.jpg
tags:
  - community call
  - events
  - multilingual
deets: |
    Meeting ID:  838-6678-3222
    Passcode: 584914
zoomurl: https://numfocus-org.zoom.us/j/83866783222?pwd=Y1k3OEJnakc3WVRISlZxQ3NTUU1yZz09
---
Open Source and Open Science, as global movements, faces language-based exclusion as most resources are in English. This affects scientists and research software engineers (RSE) working in R, particularly those who don't have English as first language. 

rOpenSci multilingual efforts aim to lower access barriers, democratize quality resources, and increase the possibilities of contributing to open software and science. We successfully piloted our Spanish-language peer review and the localization to Spanish of our comprehensive guide to software development, with Portuguese translation underway.

> On this call _Maëlle_, _Paola_ and _Elio_ will share details of the rOpenSci multilingual project. Maëlle will present the R packages that allow us to have our content in several languages while Elio and Paola will share how is the translation process and present the Translation Guide that was written to document this process. 

Author: ```suggestion

On this call _Maëlle_, _Paola_ and _Elio_ will share details of the rOpenSci Multilingual project. Maëlle will present the R packages that allow us to have our content in several languages while Elio and Paola will share how is the translation workflow and present the Translation Guide that was written to document the process. 
```
